app/messages/new-gc.tsx

```diff
@@ -0,0 +1,86 @@
+import { pocketbase, useUserStore } from "@/features/pocketbase";
+import { useMessageStore } from "@/features/pocketbase/stores/messages";
+import { Profile } from "@/features/pocketbase/stores/types";
+import { c, s } from "@/features/style";
+import { XStack, YStack } from "@/ui";
+import { Avatar } from "@/ui/atoms/Avatar";
+import MessageInput from "@/ui/messaging/MessageInput";
+import { Link, router, useLocalSearchParams } from "expo-router";
+import { useEffect, useState } from "react";
+import { View, Text, DimensionValue, TextInput } from "react-native";
+
+export default function NewGCScreen() 
+{
+  const queryParams = useLocalSearchParams();
+  const [users, setUsers] = useState<Profile[]>([]);
+  const [message, setMessage] = useState<string>('');
+  const [title, setTitle] = useState<string>('');
+  const { createConversation, sendMessage } = useMessageStore();
+  const { user } = useUserStore();
+
+  useEffect(() => {
+    const getUsers = async () => {
+      const ids = typeof queryParams.members === 'string' ? queryParams.members.split(',') : queryParams.members;
+
+      const filter = ids.map(id => `id="${id}"`).join(' || ');
+      const result = await pocketbase.collection('users').getFullList<Profile>({
+        filter: filter,
+      });
+      setUsers(result);
+    }
+    getUsers();
+  }, [])
+
+  const getUserListString = (users: Profile[]) => {
+    let s = `Chat with ${users[0].firstName }`;
+    for (let i=1; i<users.length-1; i++) {
+      s += `, ${users[i].firstName}`;
+      if (i>1) break;
+    }
+    if (users.length > 1) s += ' and ' + (users.length > 3 ? `${users.length-3} others` : users[users.length-1].firstName);
+
+    return s;
+  }
+
+  const onMessageSubmit = async () => 
+  {
+    const conversationId = await createConversation(false, user!.id, [...users.map(u=>u.id)], title);
```

Comment:
I think we can just write `users.map(u => u.id)` instead of `[...users.map(u=>u.id)]` here